### PR TITLE
fix: update dependencies (MERGE AFTER MOVING TO VERCEL)

### PR DESCRIPTION
**Pull request by nilubava**
https://github.com/prisma/docs/issues/1790

### CODE REVIEW

**Comment by tanberry** [comment]
Hi @Nilu is this one ready for merging?

File: package.json [hunk 1]
```diff
@@ -4,74 +4,74 @@
   "version": "1.0.0",
   "author": "Nilufar Bava <bava@prisma.io>",
   "dependencies": {
-    "@mdx-js/mdx": "1.6.22",
-    "@mdx-js/react": "1.6.22",
+    "@mdx-js/mdx": "^1.6.22",
+    "@mdx-js/react": "^1.6.22",
```
Comment (janpio) [inline package.json]: Renovate will instantly change all dependencies to pinned ones, so maybe avoid that additional PR/commit and just do it here already.

**Comment by janpio** [comment]
The detected visual changes by Percy are a bit sus, but I guess that the new one renders things a bit different?

- Ah, some code formatting changes quite drastically - but for the better.
- Some line height or padding also has to have changed, but minimally and it only becomes drastic in Percy because it accumulates a lot over the long files we have.
- So with all these changes super hard to say, but I think I agree with Nilu that the visual changes that Percy picked up are ok.

Not clear regarding interactive functionality of the docs though. No idea how to test that and how that could be influences as we do not really have any tests for that. Manually tested search, and that still seems to work as before.

**Comment by nilubava** [comment]
@janpio Yes, the percy changes looked ok to me :) and @tanberry Can you please confirm there is no unexpected changes in the UI - maybe just click around a bit and confirm all the pages look like before :)

**Comment by nilubava** [comment]
Deps not updated in this PR:

- Percy related packages
- @mdx-js/mdx, @mdx-js/react

File: package.json (hunk 1 above)
Comment (nilubava) [reply package.json]: 👍

File: README.md [hunk 2]
```diff
@@ -2,7 +2,8 @@
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 
-[badge-all-contributors]: https://img.shields.io/badge/all_contributors-8-orange.svg?style=flat-square
+all-contributors/add-baldyeagle
```
Comment (janpio) [inline README.md]: this looks wrong

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx [hunk 3]
```diff
@@ -192,8 +192,8 @@ model Example {
   value Json
   //    ^ field type matching the operator class
 
-  @@index([value(ops: JsonbPathOps)], type: Gin)
   //                  ^ operator class      ^ index type
+  @@index([value(ops: JsonbPathOps)], type: Gin)
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx:196]: this is wrong

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx [hunk 4]
```diff
@@ -241,9 +241,9 @@ model Example {
   value String @db.Inet
   //           ^ native type matching the operator class
 
-  @@index([value(ops: InetOps)], type: Gist)
   //                                   ^ index type
   //                  ^ operator class
+  @@index([value(ops: InetOps)], type: Gist)
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx:246]: this is wrong

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx (hunk 3 above)
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx]: this and the two changes below are wrong

**Comment by nilubava** [comment]
@janpio fixed, can you check if its good to go?

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx [hunk 5]
```diff
@@ -2864,13 +2864,16 @@ In [2.17.0](https://github.com/prisma/prisma/releases/tag/2.17.0) and later, Pri
 
 In the following example, `tags` is ignored because it refers to an invalid model (no `@id` on `Product`):
 
+<!-- prettier-ignore-start -->
+
```
Comment (janpio) [inline content/400-reference/200-api-reference/100-prisma-schema-reference.mdx:2868]:
```suggestion
<!-- prettier-ignore-start -->
```

**Comment by janpio** [comment]
Unfortunately does the CI job for Percy fail, so can not check if the layout is negatively impacted. It should not be for this to be approved and merged.

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx (hunk 5 above)
Comment (nilubava) [reply content/400-reference/200-api-reference/100-prisma-schema-reference.mdx:2868]: @janpio that was a very silly mistake 😬 Fixing it now

**Comment by nilubava** [comment]
This is a bit too polluted, so I tried a different approach to update deps. Closing this PR in favor of https://github.com/prisma/docs/pull/3778

**Pull request closed without merging**